CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## [4.0.33](https://github.com/puppetlabs/facter/tree/4.0.33) (2020-08-05)
+
+[Full Changelog](https://github.com/puppetlabs/facter/compare/4.0.32...4.0.33)
+
+### Added
+- \(FACT-2040\) Added solaris memory resolver [\#2040](https://github.com/puppetlabs/facter/pull/2040) ([sebastian-miclea](https://github.com/sebastian-miclea))
```

Comment:
The correct link is `https://github.com/puppetlabs/facter/pull/1999`

Reply:
Fixed